### PR TITLE
Important to have My Home Assistant Url in Redirect Url List (even if My Home Assistant is not used)

### PR DESCRIPTION
**Pull request by zeug-it**
It is important to have the My Home Assistant Redirect Url in the List (at developers.home-connect.com) even if only local or custom Urls are used.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/home_connect.markdown [hunk 1]
```diff
@@ -49,7 +49,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 If you don't have [My Home Assistant](/integrations/my) on your installation,
 you can use `<HOME_ASSISTANT_URL>/auth/external/callback` as the redirect URI
-instead.
+instead. But it is important to have the My Home Assistant Url in the redirect Urls list too.
```
Comment (frenck) [inline source/_integrations/home_connect.markdown:52]: That statement is incorrect. This particular paragraph is in the section "I have manually disabled My Home Assistant". In those cases, the My Home Assistant URL cannot be used.

**Comment by zeug-it** [comment]
@frenck  But it has to be configured as Redirect URL at home-connect to successful config this extension.

**Comment by frenck** [comment]
> But it has to be configured as Redirect URL at home-connect to successful config this extension.

That statement makes no sense. When My Home Assistant is disabled, the URL isn't used for redirecting. Thus configuring such a thing will be adding an unused URL. You could as well just add `https://www.google.com` 🤷 

../Frenck

**Comment by zeug-it** [comment]
I Know, but I tried only with my local URL as redirect URL and got the error that the URL is not allowed. In the return URL there is the My Homeassistant URL as redirect URL responded, even if it is not configured. I added the my homeassistant URL as additional URL, than it worked. I found a lot of people having problems with this component.

**Comment by frenck** [comment]
> I Know, but I tried only with my local URL as redirect URL and got the error that the URL is not allowed. In the return URL there is the My Homeassistant URL as redirect URL responded, even if it is not configured. I added the my homeassistant URL as additional URL, than it worked.

Sounds like Home Connect itself has requirements to the redirect URLs used. Have you looked that up and ensured your local version matched their requirements?

../Frenck

**Pull request closed without merging**